xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/MpTokenObjectAmountSerializer.java

```diff
@@ -0,0 +1,27 @@
+package org.xrpl.xrpl4j.model.jackson.modules;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdScalarSerializer;
+import org.xrpl.xrpl4j.model.transactions.MpTokenObjectAmount;
+
+import java.io.IOException;
+
+/**
+ * Custom Jackson serializer for {@link MpTokenObjectAmount}s.
+ */
+public class MpTokenObjectAmountSerializer extends StdScalarSerializer<MpTokenObjectAmount> {
```

Comment:
Clarify what this object is?

Reply:
This is for MPT amounts that aren't `MpTokenAmount`s, ie for `sfMaximumAmount`, `sfOutstandingAmount` and `sfMPTAmount`. I have a separate type for all of these because they're UInt64s, but are serialized in JSON as base 10 rather than base 16.

To your comment below about if this could be replaced by `MpTokenAmount`, I don't see how that's possible, as `MpTokenAmount` holds the issuance ID as well as the amount. In the places `MpTokenObjectAmount`s show up, there isn't necessarily an issuance ID present